### PR TITLE
Cli check tlog

### PR DESCRIPTION
**Pull request by kev-ms**
## Purpose of PR
- Check Tlog status in individual cluster

# Type of PR

- [ ] Documentation changes
- [x] Code changes
- [ ] Test changes
- [ ] CI-CD changes
- [ ] GitHub Template changes

## Issues Closed or Referenced

- Closes https://github.com/retaildevcrews/retail-edge/issues/264

### CODE REVIEW

File: bin/.kic/commands/check/tlog [hunk 1]
```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+#name: tlog
+#short: Check Virtual Tlog status on the local cluster
+
+# change to this directory
+#cd "$(dirname "${BASH_SOURCE[0]}")" || exit
+
+http "https://${AKDC_FQDN}/healthz"
```
Comment (wbreza) [inline bin/.kic/commands/check/tlog:9]: Just an option questions whether this is the correct endpoint we should be pointing to or not.

**Review by wbreza — APPROVED** [review]
LGTM

File: bin/.kic/commands/check/tlog (hunk 1 above)
Comment (bartr) [reply bin/.kic/commands/check/tlog:9]: I don't think so - I think that will hit the IMDb endpoint - not sure we have ingress for vtlog setup?

Comment (bartr) [reply bin/.kic/commands/check/tlog:9]: you could run k get po -n vtlog and check for running - I think that's what I did

**Pull request merged**